Documentation/artifact-format-v3.md

```diff
@@ -198,8 +198,11 @@ to several distinct components to be contained in one Artifact. For example,
 there may be an update to a file, and a package install contained in the same
 Artifact. For full rootfs updates, there will usually be only one payload.
 
-`type` is the type of payload contained within the image. At the moment there is
-one built-in type, `rootfs-image`, and all other strings will trigger use of
+`type` is the type of payload contained within the image. At the moment there are
+two built-in types:
+1. `rootfs-image`, 
+2. `null` (used in [empty payload artifacts](#Empty-payload-artifacts)),
```

Comment:
Perhaps put double quotes around `"rootfs-image"` so it's clear that `null` is an actual `null` value, and not a string.

Reply:
done!